### ISSUE TITLE
ReactMeteor undefined in MeteorJS package

### ISSUE THREAD

**Issue by rbaindourov** [meteor/react-packages#108]
Hello,

I've been experimenting with React and Meteor and couldn't help but notice that 
ReactMeteor.createClass throws an error when used in a package.

```
W20150911-15:54:22.195(-7)? (STDERR) ReferenceError: ReactMeteor is not defined
W20150911-15:54:22.195(-7)? (STDERR)     at Package (packages/winwinhost:reactjs-wizard/reactjs-wizard.jsx:4:10)
W20150911-15:54:22.195(-7)? (STDERR)     at packages/winwinhost:reactjs-wizard/reactjs-wizard.jsx:13:4
W20150911-15:54:22.195(-7)? (STDERR)     at packages/winwinhost:reactjs-wizard/reactjs-wizard.jsx:13:4
W20150911-15:54:22.195(-7)? (STDERR)     at /Users/robert/dev/bs-sales-assist/.meteor/local/build/programs/server/boot.js:222:10
W20150911-15:54:22.195(-7)? (STDERR)     at Array.forEach (native)
W20150911-15:54:22.195(-7)? (STDERR)     at Function._.each._.forEach (/Users/robert/.meteor/packages/meteor-tool/.1.1.4.2l3p0l++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20150911-15:54:22.195(-7)? (STDERR)     at /Users/robert/dev/bs-sales-assist/.meteor/local/build/programs/server/boot.js:117:5
```

Am I missing something in my `package.js` ?

```javascript
Package.describe({
  name: 'winwinhost:reactjs-wizard',
  version: '0.0.1',
  summary: 'Testing meteor package creation process',
  git: '',
  documentation: 'README.md'
});

Package.onUse(function(api) {
  api.versionsFrom('1.1.0.3');
  api.use('react');
  api.addFiles('reactjs-wizard.jsx');
  api.export('Wizard');
});

Package.onTest(function(api) {
  api.use('tinytest');
  api.use('winwinhost:reactjs-wizard');
  api.addFiles('reactjs-wizard-tests.js');
});
```

**Comment by stubailo** [comment]
You're still using the old API from the `reactjs:react` package. The way you use React in the official Meteor `react` package is different, and closer to normal React. Read about it here: http://react-in-meteor.readthedocs.org/en/latest/meteor-data/

In particular, you should use `React.createClass`, and `getMeteorData` instead of `getMeteorState`.

**Comment by rbaindourov** [comment]
What I did was take the example, leaderboard, and copy paste the Players and Leaderboard objects over to a package I created.

First I discovered that I needed to have
```
api.use('react');
```

in the `package.js` file and that ReactMeteor threw an error.

I figured I was not using it correctly when doing a site restricted search on google
`site:react-in-meteor.readthedocs.org "ReactMeteor"`

which didn't yield any results.

Thank you for the confirmation, I am using React.createClass and the mixin instead.
